code/modules/cargo/packs.dm

```diff
@@ -703,6 +703,38 @@
 	contains = list(/obj/item/ammo_box/magazine/wt550m9)
 	small_item = TRUE
 
+/datum/supply_pack/security/armory/wt650
+	name = "WT-650 Assualt Rifle Crate"
+	desc = "Contains two high-powered, semiautomatic rifles chambered in 5.45mm rounds. Requires Armory access to open."
+	cost = 12500
+	contains = list(/obj/item/gun/ballistic/automatic/wt650,
+					/obj/item/gun/ballistic/automatic/wt650)
+	crate_name = "assault rifle crate"
+
+/datum/supply_pack/security/armory/wt550_single
+	name = "WT-650 Assualt Rifle Single-Pack"
+	desc = "Contains one high-powered, semiautomatic rifle chambered in 5.45mm rounds. Requires Armory access to open."
+	cost = 8000
+	contains = list(/obj/item/gun/ballistic/automatic/wt650)
+	small_item = TRUE
+
+/datum/supply_pack/security/armory/wt550ammo
+	name = "WT-650 Assualt Rifle Ammo Crate"
+	desc = "Contains four 30-round magazines for the WT-650 Assualt Rifle. Each magazine is designed to facilitate rapid tactical reloads. Requires Armory access to open."
```

Comment:
```suggestion
	desc = "Contains four 30-round magazines for the WT-650 Assault Rifle. Requires Armory access to open."
```
Possibly unneeded word bloat, unless there's an actual mechanical basis for this that I missed. Remove or keep at your discretion.

Reply:
it IS a function in game, but I don't care about the description